code/modules/mob/living/simple_animal/corpse.dm

```diff
@@ -154,4 +154,18 @@
 	corpsehelmet = /obj/item/clothing/head/bearpelt
 
 /obj/effect/landmark/mobcorpse/russian/ranged
-	corpsehelmet = /obj/item/clothing/head/ushanka
\ No newline at end of file
+	corpsehelmet = /obj/item/clothing/head/ushanka
+
+
+/obj/effect/landmark/mobcorpse/russian/ranged/spetsnaz

```

Comment:
I see, but please don't add things you're not planning on using, since nobody is going to maintain this when it's not used.